src/extensions/maxi-block/maxiBlockComponent.js

```
@@ -1222,6 +1222,144 @@
 		isBlockStyleChange,
 		iframe
 	) {
+		if (iframe && iframe?.contentDocument?.body) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)